packages/vue-components/src/SiteNavButton.vue

```diff
@@ -51,15 +57,59 @@ export default {
 </script>
 
 <style scoped>
-    .toggle-site-nav-button {
+
+    .toggle-page-site-button {
         cursor: pointer;
+        height: 53px;
         left: 0;
         padding: 15px;
+        position: relative;
```

Comment:
Can try setting this to `absolute`? Because having relative position will stretch this all the way making the button clickable on the whole bar.


https://user-images.githubusercontent.com/47470981/123029408-aacd9900-d413-11eb-9f1a-e0ed3d184c45.mov